pulumi/grapl/__main__.py

```diff
@@ -320,25 +320,28 @@ def main() -> None:
             vars={},
         )
 
-        provision_vars = _get_subset(
-            local_grapl_core_job_vars,
-            {
-                "aws_env_vars_for_local",
-                "aws_region",
-                "container_images",
-                "stack_name",
-                "py_log_level",
-                "schema_properties_table_name",
-                "schema_table_name",
-                "test_user_name",
-                "user_auth_table",
-            },
-        )
+        local_provision_vars: Final[NomadVars] = {
+            "stack_name": config.STACK_NAME,
```

Comment:
I thought we removed `stack_name` in this commit?

Reply:
Hah - don't read in to my commit messages that much, this will be a squash - in this case, I did a :%s/DEPLOMENT_NAME/STACK_NAME/g but forgot to save the file before committing